### ISSUE TITLE
Remember last project used

### ISSUE THREAD

**Issue by taylordowns2000** [OpenFn/lightning#1076]
The main problem is this: https://www.loom.com/share/820f3914e3ed4d909ca9c32e77b46336

The app sends users back to projects that they haven't messed with in ages as their "default project". The routing for app.openfn.org is here:

1. https://github.com/OpenFn/Lightning/blob/main/lib/lightning_web/live/dashboard_live/index.ex#L25
2. https://github.com/OpenFn/Lightning/blob/main/lib/lightning/projects.ex#L398-L405

Please note that in order to do this well, we _may_ need to implement a projects_list, which we don't yet have a design for. (So maybe we hold off on this for now?)

**Comment by NickOpenFn** [comment]
Still required?

**Comment by taylordowns2000** [comment]
Yes, but it's only painful for people who have access to more than one project so it's not _super_ high priority.
